### PR TITLE
fix typos

### PR DESCRIPTION
**Pull request by RainRat**
fix typos

### CODE REVIEW

**Review by oddbookworm — APPROVED** [review]
Thanks! This looks great. I'm not going to miss `fronzensets` lol

**Comment by oddbookworm** [comment]
Oh yeah, you should make sure to run `python setup format` to get the lint check passing. You can install pre-commit to catch any outstanding linter errors before your commit happens
I'll go update the contributing wiki now...

File: docs/reST/c_api/color.rst [hunk 1]
```diff
@@ -26,7 +26,7 @@ Header file: src_c/include/pygame.h
 
 .. c:function:: PyObject* pgColor_New(Uint8 rgba[])
 
-   Return a new :py:class:`pygame.Color` instance for the the four element array *rgba*.
+   Return a new :py:class:`pygame.Color` instance for the four element array *rgba*.
```
Comment (gresm) [inline docs/reST/c_api/color.rst:29]:
```suggestion
   Returns a new :py:class:`pygame.Color` instance from the four element array *rgba*.
```

Comment (gresm) [reply docs/reST/c_api/color.rst:29]: Otherwise looks good to me.

File: src_c/mask.c [hunk 2]
```diff
@@ -1284,7 +1284,7 @@ cc_label(bitmask_t *input, unsigned int *image, unsigned int *ufind,
             }
             buf++;
         }
-        /* last pixel of the row, if its not also the first pixel of the row */
+        /* last pixel of the row, if it's not also the first pixel of the row */
```
Comment (oddbookworm) [inline src_c/mask.c:1287]: Lint is complaining about this line being ever so slightly too long. Run `python setup format` to update the formatting. You'll need clang-format and ruff installed

**Review by oddbookworm — CHANGES_REQUESTED** [review]
The circleCI fail from the most recent run isn't your fault, it's a freetype download issue that's been intermittently cropping up. But, the lint fail needs to be addressed before this can be merged

File: src_c/mask.c (hunk 2 above)
Comment (ankith26) [reply src_c/mask.c:1287]: `setup.py` has been removed. What you need to run now is `pre-commit run -a`. You don't need clang-format and ruff installed manually, it is handled automatically for you

**Review by ankith26 — APPROVED** [review]
LGTM, thanks for contributing! :tada:

**Comment by ankith26** [comment]
PS: ignore the new failing test, it is unrelated to your PR

**Review by oddbookworm — APPROVED** [review]
LGTM!

**Pull request merged**